apps/api/src/domain/models/shared/common-commands/delete-resource/resource-deleted.event.ts

```diff
@@ -0,0 +1,10 @@
+import { CoscradEvent } from '../../../../../domain/common';
```

Comment:
You'll also need a `ResourceDeletedEventHandler`. See `resource-published.event-handler.ts` and the associated test for inspiration.